test/jdk/com/sun/jdi/FileSocketTransportTest.java

```diff
@@ -93,6 +93,15 @@ public static void main(String[] args) throws Throwable {
             System.exit(1);
         }
 
+        if (Platform.isWindows()) {
+            try (SocketChannel channel = SocketChannel.open(StandardProtocolFamily.UNIX)) {
+                // Just see if we can create an unix domain socket on Windows.
+            } catch (UnsupportedOperationException e) {
+                // Windows version is too old to support unix domain sockets.
```

Comment:
Maybe a little `System.out.println` trace would be nice here that one can read in the output what happened.